src/transformers/models/opt/configuration_opt.py

```diff
@@ -0,0 +1,138 @@
+# coding=utf-8
+# Copyright 2022 The Metaseq Authors and The HuggingFace Inc. team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+""" OPT model configuration"""
+from ...configuration_utils import PretrainedConfig
+from ...utils import logging
+
+
+logger = logging.get_logger(__name__)
+
+OPT_PRETRAINED_CONFIG_ARCHIVE_MAP = {
+    "opt-125m": "https://huggingface.co/facebook/opt-125m/blob/main/config.json",
+    "opt-350m": "https://huggingface.co/facebook/opt-350m/blob/main/config.json",
+    "opt-1.3b": "https://huggingface.co/facebook/opt-1.3b/blob/main/config.json",
+    "opt-2.7b": "https://huggingface.co/facebook/opt-2.7b/blob/main/config.json",
+    "opt-6.7b": "https://huggingface.co/facebook/opt-6.7b/blob/main/config.json",
+    "opt-13b": "https://huggingface.co/facebook/opt-13b/blob/main/config.json",
```

Comment:
The identifiers are all missing the "facebook" namespace.

Reply:
Fixing!